### ISSUE TITLE
Make the Zashi seed phrase entry UI similar to that used by the Keystone hardware wallet

### ISSUE THREAD

**Issue by nuttycom** [zodl-inc/zodl-project#81]
## Is your feature request related to a problem? Please describe.
The UX of Keystone seed entry is really lovely: it provides a custom keyboard that only enables the keys that correspond to a valid next letter in one of the seed phrase wordlist words, and a suggestion pane that is populated only with those words. It displays 24 individual word slots; it's really slick and helps to avoid errors in seed phrase entry.

## Describe the solution you'd like
Basically provide the same sort of UI in Zashi. It's great, let's just copy it.
